### PR TITLE
Move to spark streaming

### PR DESCRIPTION
**Pull request by yifanlisc**
This PR migrates the breaking news to spark streaming from DataFlow. Note that it still uses the PubSub as data source. This one allows us to do some local testing to make sure the logic is correct (which is tricky to do with Kafka). 

Local testing and comparisons with DataFlow were done.

### CODE REVIEW

**Comment by yifanlisc** [comment]
Assigned to @sandeepshresthasc 
CC @muthusc

File: aggregator/src/main/java/snapchat/analytics/sparkstreamingjob/breakingnews/AggregateStoryStatsFn.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    public AggregateResult call(Tuple2<String, Iterable<BreakingNewsStorySnapStats>> v1) throws Exception {
+        AggregatedStats aggrStats = new AggregatedStats();
+
+        StringBuilder sb = new StringBuilder();
+        for (BreakingNewsStorySnapStats storyStat : v1._2()) {
+            ++ aggrStats.submitCount;
+            sb.append(storyStat.getSessionId() + ",");
+        }
+
+        return getAggregationOutput(v1._1(), aggrStats);
```
Comment (muthusc) [inline aggregator/src/main/java/snapchat/analytics/sparkstreamingjob/breakingnews/AggregateStoryStatsFn.java]: don't we have to update aggrStats.sessionIds before this?

Comment (yifanlisc) [inline aggregator/src/main/java/snapchat/analytics/sparkstreamingjob/breakingnews/AggregateStoryStatsFn.java]: Yes, you are right. Bad copy/paste from DataFlow job.

**Comment by yifanlisc** [comment]
Squash merge commit 4cd1a20e0c21b7a69b54f60a3cff74afc05bcd43 created by [Whaler](https://chrome.google.com/webstore/detail/whaler/kjfaikbmbbkbnjjeiambmjchclpfkedf). :whale:

**Pull request closed without merging**